docs/concepts/copilot.md

```diff
@@ -133,11 +150,27 @@ Github Copilot and Prophecy Data Copilot are both excellent tools to boost produ
 
 #### Which Datasets are accessible to Prophecy Data Copilot?
 
-In Prophecy’s 3.1 release, any Dataset, Source, or Seed within a Project are accessible when Data Copilot is called from that Project. In an upcoming release this behavior will change as follows: The assets need only to be part of the linked (Databricks, Snowflake, etc) catalog. That is, if the user can access the Datasets with their token, the Datasets should appear in the Environment tab and Copilot can access them.
+In Prophecy’s 3.1 release, any Dataset, Source, or Seed within a Project are accessible when Data Copilot is called from that Project. In an upcoming release, this behavior will change as follows: The assets need only to be part of the linked (Databricks, Snowflake, etc) catalog. That is, if the user can access the Datasets with their personal access token, then Datasets should appear in the Environment tab and Copilot can access them.
+
+#### How much will I pay for Prophecy Copilot?
+
+Copilot features are included with Prophecy’s Low-Code Spark and Low-Code SQL offerings. There would be an additional cost if you chose to use a private subscription Large Language Model (LLM) service. In this case, the cost depends on (1) the number of [word tokens](https://help.openai.com/en/articles/4936856-what-are-tokens-and-how-to-count-them) sent to the LLM provider, (2) the size of the Datasets, and (3) the number of iterations Prophecy sends requests to the LLM provider.
+
+#### Can I disable this feature for my users?
+
+For companies who run Prophecy within their [own VPC](/docs/architecture/deployment/deployment.md#private-saas-customer-vpc), admins may choose to enable or disable Data Copilot across their Prophecy Platform at deployment time.
+
+#### Can I use my own private OpenAI instance?
+
+In a future release, Administrators will have the option to connect Prophecy Data Copilot to their private subscription OpenAI.
+
+#### Can I use Data Copilot on my on-premise installation of Prophecy?
+
+Yes! Speak to your Prophecy account owner with questions.
```

Comment:
I'm not going to add the below settings in this doc because the rest of the on-prem installation instructions are not published publicly. At least we confirmed the FAQ as shown is correct. 

4 things are needed to enable copilot on customer ENV.
1. Customer needs their own openAI deployment on Azure. 
2. Copilot service needs to be added to enabled apps on `prophecycluster` CR.
3. The following flag has to be set on the configmap for ENV.
`COPILOT_ENABLED: "true" `
 4. Once the copilot-cm configMap is added in the namespace post adding of enabled app. This configMap values need to be manually filled for the customer based on the deployed ENV in step 1.
```
  AZURE_DEPLOYMENT_NAME: <goes-here>
  AZURE_OPENAI_API_KEY: <goes-here>
  AZURE_OPENAI_ENDPOINT: <goes-here>
  AZURE_OPENAI_VERSION: <goes-here>
  NUM_WORKERS: "4"
  OPENAI_API_KEY: <goes-here>
  PORT: "9016"
```